meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -16,9 +31,27 @@ SRC_URI[wl18xx-conf.md5sum] = "e0db09a1dea22b1fbcb1a5d9aa168215"
 SRC_URI[wl18xx-conf.sha256sum] = "e68e9a37995ab782faa41971704f24fd597d5abf16c47463708e90f8f080d335"
 
 do_install_append() {
+    install -d ${D}${sysconfdir}/firmware
```

Comment:
Shouldn't be needed as this is already an append, and the original do_install (from the main recipe) is already creating such folder.